modules/proc_manage-clairv4.adoc

```diff
@@ -308,3 +333,42 @@ for the image to see either all vulnerabilities or those that are fixable. The
 following figure shows information on all vulnerabilities found:
 +
 image:clair-vulnerabilities.png[See all vulnerabilities or only those that are fixable]
+
+== Clair Notifications
+<TODO>
+
+== Configuring Clair for Disconnected Environments
+
+Clair utilizes a set of components called Updaters to handle the fetching and parsing of data from various vulnerability databases.  These Updaters are set up by default to pull vulnerability data directly from the internet and work out of the box.  For customers in disconnected environments without direct access to the internet this poses a problem.  Clair supports these environments through the ability to work with different types of update workflows that take into account network isolation.  Using the `clairctl` command line utility, any process can easily fetch Updater data from the internet via an open host, securely transfer the data to an isolated host, and then import the Updater data on the isolated host into Clair itself.
+
+The steps are as follows.
+
+. First ensure that your Clair configuration has disabled automated Updaters from running.
++
+.config.yaml
+[source,yaml]
+----
+matcher:
+  disable_updaters: true
+
+----
+
+. Export out the latest Updater data to a local archive.  This requires the `clairctl` tool which can be run directly as a binary, or via the Clair container image.  Assume your Clair configuration is in `/etc/clairv4/config/config.yaml`, to run via the container image:
++
+```
+$ sudo podman run -it --rm -v /etc/clairv4/config:/cfg:Z -v /etc/clairv4/updaters:/updaters:Z --entrypoint /bin/bash {productrepo}/clair:{productminv}
+bash-4.4$ clairctl export-updaters --config /cfg/config.yaml updaters.gz
```

Comment:
```suggestion
$ podman run -it --rm -v /etc/clairv4/config:/cfg:Z -v $(pwd):/updaters:Z --entrypoint /bin/clairctl {productrepo}/clair:{productminv} export-updaters --config /cfg/config.yaml /updaters/updaters.gz
```

I don't think this is correct. As written, the output will go to `./updaters.gz`, but the pwd almost certainly isn't `/updaters`.

Reply:
good eye- yes, I ran it in the /updaters directory on the container